examples/pytorch/MNIST/example2/README.md

```diff
@@ -0,0 +1,78 @@
+## MNIST example with MLFlow
```

Comment:
cc @andreakress would you be able to help take a look at the README here for wording?